charts/scalardb/values.yaml

```diff
@@ -123,6 +123,22 @@ scalardb:
     # -- The log level of Scalar DB
     dbLogLevel: INFO
 
+  useCustomizedConfiguration:
+    # -- Use user customized database.properties. You need to create ConfigMap includes database.properties.
+    enabled: false
+    # -- ConfigMap name that includes database.properties.
+    configMapName: "scalardb-customized-config"
+    # -- Use Secret to pass the credentials as environment variable.
+    useSecret: false
```

Comment:
Is this setting necessary? Do you expect to pass credentials using a resource other than Secret?

Reply:
> Do you expect to pass credentials using a resource other than Secret?

No, I don't expect using other than Secret.
I use this configuration for the if clause only in the deployment.yaml.
So, I removed this setting.

And, I update the condition of if clause to decide to load environment variable from Secret (use `envFrom`) or not.
If the `scalardb.useCustomizedConfiguration.secretName` provided, we use `envFrom` in the deployment.yaml.